test/runtests.jl

```diff
@@ -402,18 +405,8 @@ end
 
     if is_local
         flavours = ["os", "ptp", "ata"]
-        for f in flavours
-            savefile = joinpath(@__DIR__,"mpi_df_$f.arrow")
-
-            rm(savefile, force = true) # make sure to remove any old file
-            runfile = joinpath(@__DIR__,"script_mpi_minimum_$f.jl")
-            rr = run(`$mpiexec -np 2 $juliaexec -t 1 $runfile`)
-            @test rr.exitcode == 0
-        end
-        savefiles = [joinpath(@__DIR__,"mpi_df_$f.arrow") for f in flavours]
-        dfs = [RimuIO.load_df(sf) for sf in savefiles]
-        @test reduce(==, dfs) # require equal DataFrames from seeded qmc
```

Comment:
So this test was checking whether, with identical rng seeds, the different MPI communication protocols produced the same QMC data including the random numbers. I know that the new `mpi_runtests.jl` does a lot of good new tests, but does it make sure that, say `mpi_combine_walkers()` does not accidentally drop walkers?

Reply:
Good point, I'll add a test that checks this.